makers/senior_engineer.md

```diff
@@ -7,7 +7,6 @@ Senior Engineer
 *Successfully demonstrated the job expectations for the role of an Engineer at Mozilla or equivalent role elsewhere. Additionally you should have:*
```

Comment:
I'm struggling a bit with the shift in "should have" intent between engineer and senior engineer. It's changed from qualities to actions, and it's making the language a bit stilted (especially if it's supposed to build upon the engineer description).